src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/DescriptorReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,82 @@
+Feature: Validate the reference of descriptors when creating reosurces
+
+        Background
+            Given this student exists:
+                  | Id                               | studentUniqueId | birthDate  | firstName | lastSurname |
+                  | fafbbd2024664ee786c05e899fbfe9bc | 604824          | 2010-01-13 | Traci     | Mathews     |
+              And this person exists:
+                  | personId                         | sourceSystemDescriptor |
+                  | 0063560B5AC641DBBCCE4B121C5CA37B | State                  |
+
+
+        @ignore
+        Scenario: User CAN'T create a resource when reference doesn't exist
+             When client sends a POST request to "/ed-fi/localEducationAgencies" with body
+                  """
+                  {
+                        "localEducationAgencyId": 25590100,
+                        "nameOfInstitution": "Grand Institution Test",
+                        "categories": [
+                        {
+                        "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor"
+                        }
+                        ],
+                        "localEducationAgencyCategoryDescriptor": "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor"
```

Comment:
These two descriptors are invalid, so we're getting a validation failure below instead of a reference failure. You'll need to use proper descriptor formatting, but with fake code values. For example:

`uri://ed-fi.org/EducationOrganizationCategoryDescriptor#fake` and `uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#fake`

These two descriptors do not exist and therefore will trigger the correct failure.

Reply:
Completed

Reply:
I don't see any changes related to this comment.

Reply:
Changed, I changed the response body, not the request body